docs/go/how-to-send-a-query-to-a-workflow-execution-in-go.md

```diff
@@ -0,0 +1,90 @@
+---
+id: how-to-send-a-query-to-a-workflow-execution-in-go
+title: How to send a Query to a Workflow Execution in Go
+sidebar_label: Sending Queries
+description: TODO
+tags:
+  - go
+  - developer-guide
+---
+
+You can also issue a query from your Go code using the `QueryWorkflow()` function as shown below.
```

Comment:
Are we init-capitalizing all instances of Query and Signals that are referenced in noun form?